app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+from shutil import copyfile
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
you should validate length before splitting, if length is not equal to 3, the exception will be risen here